amd/src/mod_hsuforum_loader.js

```diff
@@ -0,0 +1,44 @@
+define(['jquery'], function ($) {
+
+    /**
+     * Wait for an element before resolving a promise
+     * @param {String} querySelector - Selector of element to wait for
+     * @param {Integer} timeout - Milliseconds to wait before timing out, or 0 for no timeout
+     */
+    function waitForElement(querySelector, timeout= 0) {
+
+        const startTime = new Date().getTime();
+
+        return new Promise((resolve, reject) => {
+            const timer = setInterval(() => {
+
+                const now = new Date().getTime();
+                if(document.querySelector(querySelector)) {
+                    clearInterval(timer);
+                    resolve();
+                }else if(timeout && now - startTime >= timeout) {
+                    clearInterval(timer);
+                    reject();
+                }
+            }, 100);
+        });
+    }
+
+
+    return {
+        init: function () {
+            $('.article :input').attr('disabled', true);
+            $('.hsuforum-menu :input').attr('disabled', true);
+
+            waitForElement("body", 30000).then(function() {
```

Comment:
Is this the failsafe to remove after x amount if not removed already?

Reply:
yes